UnitTests/GitCommandsTests/GitModuleTest.cs

```diff
@@ -38,5 +41,27 @@ public void CommitCmdTests(bool amend, bool signOff, string author, bool useExpl
             var actual = _gitModule.CommitCmd(amend, signOff, author, useExplicitCommitMessage, noVerify, gpgSign, gpgKeyId);
             StringAssert.AreEqualIgnoringCase(expected, actual);
         }
+
+        [Test]
+        public void ParseGitBlame()
+        {
+            var path = Path.Combine(TestContext.CurrentContext.TestDirectory, @"TestData/README.blame");
```

Comment:
`EmbeddedResourceLoader` is an alternative to loading files.

Reply:
This approach was lifted from `PatchManagerTest`. Looks like it exists in `TestDiffLineNumAnalyzer` too.

Reply:
No worries, was pure FYI